test/gnina/test_flexmin.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
```

Comment:
**This test does nothing!** It has to be removed, or a comparison with the expected minimised structure has to be added.

Reply:
you might want to test whether the expected residues have moved and whether the CNN score has improved without doing a full structure comparison, which will depend on a particular CNN model which is subject to change. you could also test individual functions, e.g. if you populate the diff blob with specific values, do residues move the way you'd expect (which also won't depend on a specific CNN model, since you'll be filling in the diff)